client/my-sites/domains/components/domain-warnings/pending-gsuite-tos-notice.jsx

```diff
@@ -70,9 +70,10 @@ class PendingGSuiteTosNotice extends React.PureComponent {
 
 	getExclamation( severity ) {
 		const { translate } = this.props;
+
 		const translationOptions = {
-			context: 'Beginning of Gapps pending account notice',
-			comment: "Used as an exclamation in Gapps' pending account notice",
+			context: 'Beginning of G Suite pending account notice',
```

Comment:
Should we not change this to "Google" or "Google Workspace"? "G Suite" has a pretty limited lifespan at this stage!

Reply:
I wasn't sure about that one, I'll update it.